src/share/poudriere/jail.sh

```diff
@@ -208,6 +208,30 @@ hook_stop_jail() {
 	fi
 }
 
+update_pkgbase() {
+	local make_jobs
+	local destdir="${JAILMNT}"
+
+	if [ ${JAIL_OSVERSION} -gt 1100086 ]; then
+		make_jobs="${MAKE_JOBS}"
+	fi
+
+	msg "Starting make update-packages"
+	${MAKE_CMD} -C "${SRC_BASE}" ${make_jobs} update-packages \
+		    DESTDIR=${destdir} REPODIR=${POUDRIERE_DATA}/images/${JAILNAME}-repo ${MAKEWORLDARGS}
```

Comment:
Should quote all of this (and later)
`DESTDIR="${destdir}" REPODIR="..."`
Don't quote `${MAKEWORLDARGS}` though of course.